protobuf_to_dict/convertor.py

```diff
@@ -226,13 +227,18 @@ def _dict_to_protobuf(pb, value, type_callable_map, strict, ignore_none):
                 else:
                     pb_value.append(item)
             continue
+
         if isinstance(input_value, datetime.datetime):
             input_value = datetime_to_timestamp(input_value)
             # Instead of setattr we need to use CopyFrom for composite fields
             # Otherwise we will get AttributeError:
             #   Assignment not allowed to composite field “field name” in protocol message object
             getattr(pb, field.name).CopyFrom(input_value)
             continue
+        elif field.name == 'fetched_by':
```

Comment:
@ll10524 I don't think `fetched_by` belongs to this repo. This is an open source library. Fair's internal business logic shouldn't be here. However what you can do is maybe add a `use_date_parser_for_fields` parameter to `dict_to_protobuf` so that field names that are in the set of names that are passed use date parser.

So in that case when this library is used in your micro service, you will pass `use_date_parser_for_fields={'fetched_by'}`

Reply:
Good suggestion. Thanks.